CWE/decisions/Superstate - african_union.txt

```diff
@@ -215,7 +215,15 @@ political_decisions = {
 		potential = {
 			NOT = { has_country_flag = hide_superstate }
 			tag = XAU
-			any_neighbor_country = { has_country_flag = xau_ambition in_sphere = THIS NOT = { has_country_modifier = rejected_union } }
+			OR = {
+				any_neighbor_country = { has_country_flag = xau_ambition in_sphere = THIS NOT = { has_country_modifier = rejected_union } }
+				MAD = {
+					exists = yes
+					has_country_flag = xau_ambition
+					in_sphere = THIS
```

Comment:
Must have this check instead to prevent XAU performing a diplo steal of someone else’s puppet: 

OR = { vassal_of = THIS in_sphere = THIS }

Reply:
Alright, do any of the ones I didn't edit have that same thing missing as well?

Reply:
Alright, should that be in the regular check as well too? I don't think its on the regular check, and I don't think the decisions I didn't edit have it either.

Reply:
Alright!